apps/communications/contacts/js/export/contacts_exporter.js

```diff
@@ -156,20 +155,10 @@ window.ContactsExporter = function ContactsExporter(theStrategy) {
   };
 
   //
-  // Based on the strategy configure the progress display to show a
-  // determinative or indeterminate ui depending on the strategy
-  //
-  var _configureProgress = function _configureProgress() {
-    determinativeProgress =
-      strategy.hasDeterminativeProgress !== undefined &&
-      strategy.hasDeterminativeProgress();
-  };
-
-  //
   // Shows the progress dialog based on the
   //
   var _displayProgress = function _displayProgress() {
-    var progressClass = determinativeProgress ? 'progressBar' : 'spinner';
+    var progressClass = true;
```

Comment:
Yes, We have to show a progress bar only as per current UX. That is the reason we use "true". It's working same as expected in UX. In next commit I will remove the "progressClass" variable.